app/logic/events.py

```diff
@@ -113,23 +113,24 @@ def attemptSaveMultipleOfferings(eventData, attachmentFiles = None):
     failedSavedOfferings = []
     allSavesWereSuccessful = True
     
-    # Creates a shared multipleOfferingId for all offerings to have
-    multipleOfferingId = calculateNewMultipleOfferingId()
+    seriesId = calculateNewSeriesId()
 
     # Create separate event data inheriting from the original eventData
-    multipleOfferingData = eventData.get('multipleOfferingData')
+    seriesData = eventData.get('seriesData')
+    isRepeating = bool(eventData.get('isRepeating'))
```

Comment:
should we be using `preprocessEventData`? It's already doing the cleanup of these fields, right?